test/jasmine/tests/svg_text_utils_test.js

```diff
@@ -105,6 +105,21 @@ describe('svg+text utils', function() {
             assertAnchorLink(node, 'mailto:support@plot.ly');
         });
 
+        it('wrap XSS attacks in href', function() {
```

Comment:
Looks good to me!  Let me know when it's on prod and I'll play with it a bit to be extra sure.